tailwind.config.ts

```diff
@@ -6,6 +6,7 @@ const config: Config = {
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/components/**/*.{js,ts,jsx,tsx,mdx}",
     "./src/app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/common/**/*.{js,ts,jsx,tsx,mdx}",
   ],
```

Comment:
why we need to add this path?

Reply:
@nadirhussain786 some times tailwind classes not work , so we add path of the `directory` to apply tailwind classes , in this line i am give the path of `common` folder